src/providers/theme/theme.ts

```diff
@@ -0,0 +1,30 @@
+import { PaletteMode, ThemeOptions } from '@mui/material';
+import { getPalette } from './colorPalette';
+
+export enum ThemeMode {
+  light = 'light',
+  dark = 'dark',
```

Comment:
I checked the TS docs and they name enum fields with capitalized letter. Let's do it this way.
[](https://www.typescriptlang.org/docs/handbook/enums.html)
```
export enum ThemeMode {
  Light = 'light',
  Dark = 'dark',
}
```